app/src/main/res/values-en/strings.xml

```diff
@@ -0,0 +1,40 @@
+<resources>
+    <string name="main_activity_description">Chat with your mentors.</string>
+
+    <string name="login">Login</string>
+    <string name="chat_with_mentor">Chat with a mentor</string>
+
+    <!-- NOTIFICATION SERVICE -->
+    <string name="notification_title">You have a new notification</string>
+
+    <!-- MAIN ACTIVITY -->
+    <string name="error_not_connected">You must first login!</string>
+    <string name="error_authentication_canceled">Login aborted</string>
+    <string name="error_no_internet">No internet connection!</string>
+    <string name="error_unknown_error">An unknown error occurred</string>
+    <string name="connection_succeed">Login successful!</string>
+    <string name="button_login_text_logged">View your profile</string>
+    <string name="button_login_text_not_logged">Login</string>
+
+    <!-- LOGIN ACTIVITY -->
+    <string name="toolbar_title_login_activity">Your Profile</string>
+    <string name="text_input_username">Username</string>
+    <string name="text_view_email">Email address:</string>
+    <string name="button_update_account">Update username</string>
+    <string name="button_delete_account">Delete my account</string>
+    <string name="button_sign_out_account">Logout</string>
+    <string name="info_no_email_found">No email address found</string>
+    <string name="info_no_username_found">No username found</string>
+    <string name="popup_message_confirmation_delete_account">Are you sure you wish to delete your account?</string>
+    <string name="popup_message_choice_yes">Yes!</string>
```

Comment:
I'd take the exclamation mark out here, I think it looks strange for one word to have punctuation and the other not to, especially since they'll be side by side.